staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/custom.go

```diff
@@ -140,7 +140,7 @@ func (cache *checkers) getCheckerInternal(rt reflect.Type, parent *path) (c chec
 	var wg sync.WaitGroup
 	wg.Add(1)
 	defer wg.Done()
-	c = checker{
+	lazy := checker{
```

Comment:
lazy is a strange name. Call it placeholder?

Reply:
Thanks, renamed.